### PR TITLE
Add PauseMenuDataMgr::x_28 - types are incorrect

### PR DESCRIPTION
**Pull request by savage13**
Add PauseMenuDataMgr::x_28
Note: Types are incorrect

---
This change is [Reviewable](https://reviewable.io/reviews/zeldaret/botw/87)

### CODE REVIEW

File: src/Game/UI/uiPauseMenuDataMgr.h [hunk 1]
```diff
@@ -366,6 +367,9 @@ class PauseMenuDataMgr {
     void equipWeapon(PouchItem* weapon);
     void unequip(PouchItem* item);
 
+    // FIXME: types
+    int x_28(void* unk, int a3, PouchItem* item);
```
Comment (leoetlino) [inline src/Game/UI/uiPauseMenuDataMgr.h:371]:
```suggestion
    bool useItemFromRecipeAndSave(void* recipe, int multiplier, PouchItem* item);
```
avoid placeholder names as much as you can

File: src/Game/UI/uiPauseMenuDataMgr.h [hunk 2]
```diff
@@ -400,6 +404,9 @@ class PauseMenuDataMgr {
     sead::OffsetList<PouchItem>& getItems() { return mItemLists.list1; }
     const sead::OffsetList<PouchItem>& getItems() const { return mItemLists.list1; }
 
+    // FIXME: types
+    int pouchUseFromRecipe(Lists* lists, void* unk, int a4, PouchItem* a5);
```
Comment (leoetlino) [inline src/Game/UI/uiPauseMenuDataMgr.h:408]:
```suggestion
    bool useItemFromRecipe(Lists* lists, void* recipe, int multiplier, PouchItem* item);
```

**Review by leoetlino — CHANGES_REQUESTED** [review]

**Review by savage13 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/zeldaret/botw/87)* status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @leoetlino)

___
*[src/Game/UI/uiPauseMenuDataMgr.h, line 371 at r1](https://reviewable.io/reviews/zeldaret/botw/87#-MwqPByxoiWYCO3dewGs-r1-371:-MwqSLU9CjPUKXzkdNBP:bb6un1j) ([raw file](https://github.com/zeldaret/botw/blob/8460270884e0b22b4e0281f11a1721f931d3e8e4/src/Game/UI/uiPauseMenuDataMgr.h#L371)):*
<details><summary><i>Previously, leoetlino (Léo Lam) wrote…</i></summary><blockquote>

```suggestion
    bool useItemFromRecipeAndSave(void* recipe, int multiplier, PouchItem* item);
```
avoid placeholder names as much as you can, especially when the names can be guessed/deduced
</blockquote></details>

Done
___
*[src/Game/UI/uiPauseMenuDataMgr.h, line 408 at r1](https://reviewable.io/reviews/zeldaret/botw/87#-MwqPByxoiWYCO3dewGs-r1-408:-MwqT0iq4eezqYgAjykh:b-896fix) ([raw file](https://github.com/zeldaret/botw/blob/8460270884e0b22b4e0281f11a1721f931d3e8e4/src/Game/UI/uiPauseMenuDataMgr.h#L408)):*
<details><summary><i>Previously, leoetlino (Léo Lam) wrote…</i></summary><blockquote>

```suggestion
    bool useItemFromRecipe(Lists* lists, void* recipe, int multiplier, PouchItem* item);
```
</blockquote></details>

Done.

**Review by savage13 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/zeldaret/botw/87)* status: all files reviewed, 2 unresolved discussions (waiting on @leoetlino)

___
*[src/Game/UI/uiPauseMenuDataMgr.h, line 371 at r1](https://reviewable.io/reviews/zeldaret/botw/87#-MwqPByxoiWYCO3dewGs-r1-371:-MwqaRcTAUK8iujg0OW9:b-896fix) ([raw file](https://github.com/zeldaret/botw/blob/8460270884e0b22b4e0281f11a1721f931d3e8e4/src/Game/UI/uiPauseMenuDataMgr.h#L371)):*
<details><summary><i>Previously, leoetlino (Léo Lam) wrote…</i></summary><blockquote>

This should return bool.
</blockquote></details>

Done.
___
*[src/Game/UI/uiPauseMenuDataMgr.h, line 408 at r1](https://reviewable.io/reviews/zeldaret/botw/87#-MwqPByxoiWYCO3dewGs-r1-408:-MwqaRHg4RF6I3mid6R8:b-896fix) ([raw file](https://github.com/zeldaret/botw/blob/8460270884e0b22b4e0281f11a1721f931d3e8e4/src/Game/UI/uiPauseMenuDataMgr.h#L408)):*
<details><summary><i>Previously, leoetlino (Léo Lam) wrote…</i></summary><blockquote>

bool
</blockquote></details>

Done.

**Review by leoetlino — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/zeldaret/botw/87)* status: all files reviewed (commit messages unreviewed), 1 unresolved discussion (waiting on @savage13)

___
*[src/Game/UI/uiPauseMenuDataMgr.h, line 408 at r1](https://reviewable.io/reviews/zeldaret/botw/87#-MwqPByxoiWYCO3dewGs-r1-408:-Mwqc4zI9topbn49ayFn:b-rxuspq) ([raw file](https://github.com/zeldaret/botw/blob/8460270884e0b22b4e0281f11a1721f931d3e8e4/src/Game/UI/uiPauseMenuDataMgr.h#L408)):*
<details><summary><i>Previously, savage13 wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Not done :P

**Review by leoetlino — APPROVED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/zeldaret/botw/87)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @savage13)

**Pull request merged**